roles/eclipse/vars/main.yml

```diff
@@ -4,6 +4,7 @@ eclipse:
   # Ensure the URL does not specify a mirror and that &r=1 is at the end, which
   # directly links to the file and not the web page with a download button
   url: 'https://www.eclipse.org/downloads/download.php?file=/technology/epp/downloads/release/photon/R/eclipse-java-photon-R-linux-gtk-x86_64.tar.gz&r=1'
+  urlbck: 'https://www.eclipse.org/downloads/download.php?file=/oomph/epp/photon/R/eclipse-inst-linux64.tar.gz&mirror_id=504'
```

Comment:
We need the URL for Eclipse for Java Developers, not the installer, so it should have a URL with `eclipse-java-photon-R-linux-gtk-x86_64.tar.gz` somewhere in it.

Additionally, can this variable use a longer name without an abbreviation like `backup_url` or `url_backup`?

Reply:
ah yes, I'll get that set up

Reply:
In the looping rewrite, I was encouraging prefixing variable names with the role name, as they're all global.